evap/staff/tests/test_views.py

```diff
@@ -870,29 +886,22 @@ def test_import_invalid_contributors_file(self):
         self.assertContains(reply, 'Errors occurred while parsing the input data. No data was imported.')
         self.assertNotContains(reply, 'Import previously uploaded file')
 
-        self.assertEqual(UserProfile.objects.count(), original_user_count)
+    def test_import_contributors_warning_handling(self):
+        # Tests whether warnings given from the importer are displayed
```

Comment:
please use `""" comment goes here """` style for comments describing the whole function.